### DISCUSSION TITLE
Challenging the standard cosmological model - discussion meeting 2024-4-15

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#176] [Cosmology Models]
Scientific discussion meeting organised by Professor James Binney FRS, Dr Roya Mohayaee, Professor John Peacock FRS and Professor Subir Sarkar.
[https://royalsociety.org/science-events-and-lectures/2024/04/cosmological-model/](https://royalsociety.org/science-events-and-lectures/2024/04/cosmological-model/)

2024 April 15-16; Both in person and online attendance available

**Comment by budrap00** [comment]
From the overview:

"Is the universe simple enough to be adequately described by the standard ΛCDM cosmological model which assumes the isotropic and homogeneous Friedmann-Lemaître-Robertson-Walker metric?"

The more that question is asked, the more obvious it will be that the answer is no.

**Reply by mikehelland** [reply]
Or is it the other way, the universe is simpler than LCDM?

**Comment by clmarchesan** [comment]
It could be a way to present our models if we send them via email.

**Reply by RedshiftDrift** [reply]
This is organized by the Royal Society, send your papers to the [Philosophical Transactions of the Royal Society A](https://royalsocietypublishing.org/journal/rsta) if you want 'them' to look at your model.

**Reply by clmarchesan** [reply]
Many thanks bye

**Comment by bgreenman** [comment]
Great conference idea, helps promote alternatives to standard models

**Comment by OlliSant** [comment]
I have not enough competence to present my theory there. Is there anybody who has a similar model and has competence to present it?

 It is simpler than Lambda model. 4D, and Q as diameter, eternal time. Different extrapolation as in BB. GR chanced to 4D, as in Suntolas DU, but no expansion. 

The theory of the Universe must say what is outside the observable universe too. There is the web of galaxies eternally and everywhere. 

The universe, the whole universe has no outside. Such place does not exists. And there is no beginning. The local universe goes like BB, but no expansion there either, except in the beginning of the Local Universe, the local big bang.

Can be tested in astronomy: there should be visible different beginnings, different galaxy populations.

**Reply by genotics** [reply]
Hi Oli, Toivo had a very succinct expression to describe the infinite and eternal Universe: Every thing evolves, but the whole does not evolve. He characterized the origin and expansion theory as hyperevolutionism.

**Reply by OlliSant** [reply]
Very good! Everything evolves, but the whole does not evolve. Origin and expansion are impossible for the whole, the universe. Is it then a logical truth? 

What about my invention, that it is time, what is coming more, not matter, not space. It's only a little complicated because they are relative all four.

**Reply by bgreenman** [reply]
genotics could be right

**Reply by OlliSant** [reply]
Universe is the web of galaxies everywhere and always.

If this can be proved right, it is a big invention. New theory. Totally new cosmology. The deep field images and the animations point to this direction.

**Reply by bgreenman** [reply]
Great start

**Comment by RedshiftDrift** [comment]
Challenges of Modern Cosmology 2024 ([CMC2024](https://sites.google.com/view/cmc2024-challengesofmoderncosm)) by Elena Asencio and Pavel Kroupa

CMC2024 is an online discussion panel intended for listening, sharing and learning about challenges of modern cosmology and alternative theories. It will take place on the 18th of January 2024 from 12:00 to 17:00 (CET) and it will be publicly streamed on youtube. The panel will be divided in three sessions: current problems of modern cosmology, modified cosmology and gravity theories, and standard cosmological and gravitational tests. Each of these sessions will include from two to three 10 minutes talks followed by a 20 minutes discussion on the corresponding topic of the talk.

In order to join the discussion panel, registration is still possible until the 10th of January. The discussion can also be followed online without registration by searching for "CMC2024" on youtube or through the link that will be shared on the [CMC2024 webpage](https://sites.google.com/view/cmc2024-challengesofmoderncosm) 24h before the event.

[https://darkmattercrisis.wordpress.com/2023/12/26/87-two-upcoming-on-line-meetings-on-the-cosmological-crisis/](https://darkmattercrisis.wordpress.com/2023/12/26/87-two-upcoming-on-line-meetings-on-the-cosmological-crisis/)

**Reply by RedshiftDrift** [reply]
The [CMC-2 discussion panel](https://sites.google.com/view/cmc2024-challengesofmoderncosm/start) videos of the presentations are on YouTube: [Part I](https://youtu.be/ErHB4Ckzgu8?t=10), [Part II](https://youtu.be/0sLCr-n8u5U?t=346).

My [poster is available here](https://sites.google.com/view/cmc2024-challengesofmoderncosm/cmc2-posters#h.loz3apfyl8tm). (Eric Lerner's slides are also there.)
The list I give in Section "VI. Misconceptions about Tired-Light" can help @sahil5d [debunk “Errors in Tired Light Cosmology”](https://github.com/orgs/a-cosmology-group/discussions/262#discussion-7102855), but it would be better, if possible, to debunk without using a specific model.

A more detailed calculation of the STz process is given in a pre-print of the paper submitted to the MG17 conference.  It is [posted on arXiv](https://arxiv.org/abs/2410.02036).  This will help @HanDeBruijn understand [the formula displayed here](https://github.com/orgs/a-cosmology-group/discussions/277#discussioncomment-10991729).

[My CMC-2 talk is here](https://www.youtube.com/watch?v=0sLCr-n8u5U&t=8279s).  Instead of presenting equations in five(+3) minutes, I decided to give a qualitative description through a familiar analogy.  I'm not sure this is a better approach.  (I received 'positive' feedback from three people, but 'confused' feedback from another.)

**Reply by sahil5d** [reply]
_"The energy that's been taken out from these photons is now in the form of heat, so I've represented the red dots here which are electrons that are heated by the process."_ - @RedshiftDrift 
https://www.youtube.com/live/0sLCr-n8u5U?si=RVEUcyA2-fAqSeRW&t=8686

As your model says that an electron gets heated, that means the electron increases velocity, right? Can you show conservation of energy and momentum? Specifically, a *before* and *after* for one complete interaction?

**Comment by clmarchesan** [comment]
Hi Louis,

I want to congratulate you and Eric on your presentation. Regarding the
Tired Light theory, I recall there being an issue with the redshift
spectrum (though I don't remember the details), and solving that problem is
a significant step forward. Unfortunately, my English isn't fluent enough,
so I usually don't participate in online sessions.

I wish you all the best,
Claudio

**Reply by RedshiftDrift** [reply]
Thanks Claudio.  Let me know if you remember the specific issue with the redshift spectrum.
I've been checking all the possible issues related to redshift, I would like to know if the one you have in mind is resolved by STz.

**Comment by clmarchesan** [comment]
All I have is a vague memory concerning the CMB, whose blackbody spectrum
would not be easily explained by interactions with other particles. I don't
know anything else.
